2/contrib/jenkins/install-plugins.sh

```diff
@@ -194,6 +194,13 @@ function download() {
         fi
 
         resolveDependencies "$plugin"
+    else
+        lockFile=$(getLockFile "$plugin")
+        lockedVersion=$(cat $lockFile)
+        echo "Plugin $plugin locked to version $lockedVersion, ignoring, requested version $version"
+        if versionLT "${lockedVersion}" "${version}"; then
```

Comment:
what will we do with the warning message?
and during CI, we will not check it for sure. Should we fail in case of warning ?

Reply:
The problem is that sometimes the versions are like `2.6.5` and sometimes they switch to `1123.456.asdflasdf`, and the script does not account for `2.6.5` could be newer than `1123.456.asdflasdf`, so if we show the warnings when someone updates the `base-plugins.txt` and then runs the install script, they can sort it out manually and update the locked version if needed, which it probably is. But specific plugins we want locked to a newer version that has CVE fixes.